website/sidebars.js

```diff
@@ -246,7 +246,7 @@ module.exports = {
                         'user_guide/activation_strategy',
                         'advanced/archived_toggles',
                         'advanced/audit_log',
-                        'advanced/impression_data',
+                        'advanced/impression-data',
```

Comment:
I've changed it from an underscore to a hyphen. As of today, there's no clear standard to follow in this code base (neither for URLs nor for file names), but I'd like to standardize everything on using hyphens, so I've changed the URL here. That'll need to be updated in the front end too.

Reply:
Can we keep it as underscore and revisit this later? We've started merging the refactorings into the frontend at this point.